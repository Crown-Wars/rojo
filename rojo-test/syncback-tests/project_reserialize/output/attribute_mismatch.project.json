{
  "name": "attribute_mismatch",
  "tree": {
    "$className": "Folder"
  }
}