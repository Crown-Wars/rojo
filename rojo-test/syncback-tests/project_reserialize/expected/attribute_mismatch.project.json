{
  "name": "attribute_mismatch",
  "tree": {
    "$className": "Folder",
    "$attributes": {
      "foo": "bar"
    }
  }
}